hs_core/hydroshare/utils.py

```diff
@@ -163,10 +163,23 @@ def is_federated(homepath):
     Returns:
     True is the selected file indicated by homepath is from a federated zone, False if otherwise
     """
-    irods_storage = IrodsStorage('federated')
-    # if HS WWW iRODS proxy user can list homepath, homepath is federated; otherwise, it is not
-    # federated
-    return irods_storage.exists(homepath)
+    homepath = homepath.strip()
+    homepath_list = homepath.split('/')
+    if homepath_list[1]:
```

Comment:
Would be good to give an example of ```homepath``` value here in which case the if statement would be true.